packages/icons/icons.json

```diff
@@ -4093,5 +4093,19 @@
         "tags": "shield, new, add, marking",
         "group": "miscellaneous",
         "codepoint": 62281
+    },
+    {
+        "displayName": "group",
+        "iconName": "group",
```

Comment:
discussed over Slack, please rename this to be more specific, either:

1. "widget-group"
2. "widget-set"
3. "set-item"
4. "group-item"
5. "item-in-group"